doc/classes/Dictionary.xml

```diff
@@ -294,6 +294,20 @@
 			<param index="1" name="overwrite" type="bool" default="false" />
 			<description>
 				Adds entries from [param dictionary] to this dictionary. By default, duplicate keys are not copied over, unless [param overwrite] is [code]true[/code].
+				[codeblock]
+				var dict = { "item": "sword", "quantity": 2 }
+				var other_dict = { "quantity": 15, "color": "silver" }
+
+				# Overwriting of existing keys is disabled by default.
+				# `Dictionary.merge()` modifies `dict` and does not return a value, so we need to use it on its own line.
+				dict.merge(other_dict)
+				print(dict)  # { "item": "sword", "quantity": 2, "color": "silver" }
+
+				# With overwriting of existing keys enabled.
+				dict.merge(other_dict, true)
+				print(dict)  # { "item": "sword", "quantity": 15, "color": "silver" }
+				[/codeblock]
+				[b]Note:[/b] [method merge] is [i]not[/i] recursive. Nested dictionaries are considered as keys that can be overridden or not depending on the value of [param overwrite], but they will never be merged together.
```

Comment:
We should probably use "overwritten" over "overridden", no? Though I'm not entirely sure if "overwrite" is correct name for the argument here.